packages/core/core.ts

```diff
@@ -74,8 +74,8 @@ export async function initFuzzing(options: Options) {
 	registerGlobals();
 	registerInstrumentor(
 		new Instrumentor(
-			options.includes,
-			options.excludes,
+			options.includes === undefined ? [] : options.includes,
```

Comment:
This check is probably not needed, as `Instrumentor` defines default values for both parameters.